tf/modules/oonith_service/main.tf

```diff
@@ -220,3 +220,22 @@ resource "aws_acm_certificate_validation" "oonith_service" {
     aws_route53_record.oonith_service
   ]
 }
+
+resource "aws_route53_record" "oonith_service_alias" {
+  count = length(var.alternative_names)
+
+  zone_id = var.dns_zone_ooni_io
+  name    = var.alternative_names[count.index].name
+  type    = var.alternative_names[count.index].type
+
+  alias {
+    name                   = aws_alb.oonith_service.dns_name
+    zone_id                = aws_alb.oonith_service.zone_id
+    evaluate_target_health = true
+  }
+}
+
+resource "aws_acm_certificate_validation" "oonith_service_alias" {
+  certificate_arn         = aws_acm_certificate.oonith_service.arn
+  validation_record_fqdns = [for record in aws_route53_record.oonith_service_alias : record.fqdn]
```

Comment:
I think you should add here also the `depends_on` `aws_route53_record. oonith_service_alias` and another block which writes the validation records, like is done in `resource "aws_route53_record" "oonith_service_validation"`.

We can possibly re-use the existing `oonith_service_validation` block by modifying the `for_each` block to iterate on top of `aws_acm_certificate.oonith_service.domain_validation_options + aws_acm_certificate_validation. oonith_service_alias.domain_validation_options`